src/main/java/detail/tasks/ComputeDescriptiveStatisticsTask.java

```diff
@@ -0,0 +1,90 @@
+package detail.tasks;
+
+import detail.JStateMessage;
+import detail.compute.DescriptiveStatistics;
+import detail.datasets.IDataSet;
+import detail.datasets.IDataSetContainer;
+import detail.datasets.TableDataSet;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+/**
+ * Compute the descriptive statistics of the columns from the given
+ * data set
+ */
+public class ComputeDescriptiveStatisticsTask extends TaskBase {
+
+    public ComputeDescriptiveStatisticsTask(String name, IDataSet dataSet, String... names){
```

Comment:
what do the ... mean?

Reply:
yes I need to comment this. Wrote it yesterday on the fly. I need sth to distinguish that tasks submitted to the work pool so ```name``` is the task name which should be globally unique.
```dataSet``` is the data set instance for which the specific computation was requetsted. The
```ComputeDescriptiveStatisticsTask``` is meant to compute descriptive statistics for the columns specified in the list ```names``` of the given ```dataSet``` instance